### ISSUE TITLE
Kitti second testing on validation set not working/confusion

### ISSUE THREAD

**Issue by makeradventures** [open-mmlab/OpenPCDet#1406]
Hello,
I am training second on the kitti dataset. When the training script completes. A tensorboard_val folder is created inside a folder "/kitti_models/second/default/eval/eval_with_train". This evaluation is done on the training set after 80 epochs.

Then I run the test script testing the results on epoch 80. Here I would expect to get testing/validation results, which are slightly worse then the evaluation on the training set. However they are exactly the same, suggesting that testing is also done on the training dataset! For example eval_with_train results for Car_3d/easy_R40 are 89.02 and in testing they are 89.02 too. All the values are the same!

How do I test on a validation dataset using second and kitti? Do I have to create a train-val split myself if so how?

Thanks in advance!
